p2p/src/channels/p2p_channels_effectful_effects.rs

```diff
@@ -60,21 +60,18 @@ impl P2pChannelsEffectfulAction {
                 peer_id,
                 pub_key,
                 offer,
-            } => {
-                match store.service().encrypt(&pub_key, offer.as_ref()) {
-                    Err(_) => {
-                        // todo!("Failed to encrypt webrtc offer. Handle it.")
-                    }
-                    Ok(offer) => {
-                        let message = SignalingDiscoveryChannelMsg::DiscoveredAccept(offer);
-                        store.service().channel_send(
-                            peer_id,
-                            super::MsgId::first(),
-                            message.into(),
-                        );
-                    }
+            } => match store.service().encrypt(&pub_key, offer.as_ref()) {
+                Err(_) => {
+                    // TODO: handle
+                    bug_condition!("Failed to encrypt webrtc offer")
```

Comment:
@0xMimir sorry lets do as @binier suggests and use regular logging, because if a malicious peer can trigger this then this isn't an internal bug.